homeassistant/components/rfxtrx/translations/en.json

```diff
@@ -0,0 +1,34 @@
+{
+    "options": {
+        "step": {
+            "prompt_options": {
+                "data": {
+                  "debug": "Enable debugging",
+                  "automatic_add": "Enable automatic add",
+                  "device_id": "Enter device id to add",
```

Comment:
This is an event code right? So needs re-woring. Also the key should probably reflect that so we don't mix it up with our device_id tuple.